src/types/form.ts

```diff
@@ -336,6 +336,10 @@ export interface IFormModel extends Model<IFormSchema> {
     userEmail: IUserSchema['email'],
   ): Promise<AdminDashboardFormMetaDto[]>
 
+  getFormsOwnedByUserId(
```

Comment:
semantically, suggest `retrieveFormsOwnedByUserId` instead of `get`

Reply:
Done!